stardist/scripts/predict2d.py

```diff
@@ -0,0 +1,90 @@
+"""
+
+Command line script to perform prediction in 2D
+
+"""
+
+
+import os
+import sys
+import numpy as np
+from tqdm import tqdm
+import json 
+import argparse
+import pprint 
+import pathlib
+import warnings
+
+def main():
+    parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter, description="""
+Prediction script for a 2D stardist model, usage: stardist-predict -i input.tif -m model_folder_or_pretrained_name -o output_folder
+
+""")
+    parser.add_argument("-i","--input", type=str, nargs="+", required=True, help = "input file (tiff)")
+    parser.add_argument("-o","--outdir", type=str, default='.', help = "output directory")
+    parser.add_argument("--outname", type=str, nargs="+", default='{img}.stardist.tif', help = "output file name (tiff)")
+
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument('-m', '--model', type=str, default=None, help = "model folder / pretrained model to use")
+    parser.add_argument("--axes", type=str, default = None, help = "axes to use for the input, e.g. 'XYC'")
+    parser.add_argument("--n_tiles", type=int, nargs=2, default = None, help = "number of tiles to use for prediction")
+    parser.add_argument("--pnorm", type=float, nargs=2, default = [3,99.8], help = "pmin/pmax to use for normalization")
```

Comment:
Different from the "default" percentiles we normally use.

Reply:
feel free to change!